stubs/docopt/METADATA.toml

```diff
@@ -1,2 +1,10 @@
 version = "0.6.*"
 upstream_repository = "https://github.com/docopt/docopt"
+no_longer_updated = true
+obsolete_since = "0.8.1"
```

Comment:
obsolete_since is strictly intended for packages that added their own `py.typed` file. It will add the following text:

> *Note:* The `{distribution}` package includes type annotations or type stubs
since version {obsolete_since}. Please uninstall the `{stub_distribution}`
package if you use this or a newer version.

This would be wrong in this case, since docopt-ng added the type annotations.

Reply:
```suggestion
no_longer_updated = true
```